dbld/images/devshell.dockerfile

```diff
@@ -6,3 +6,4 @@ RUN /helpers/dependencies.sh enable_dbgsyms_on_ubuntu
 RUN /helpers/dependencies.sh install_perf
 
 RUN /helpers/dependencies.sh install_apt_packages
+RUN /helpers/dependencies.sh install_pip_packages
```

Comment:
this should have already been run in the parent image, devshell currently just "extends" the syslog-ng-ubuntu-bionic image.

Reply:
There are `devshell` only packages in the `pip_packages.manifest` file: 
https://github.com/syslog-ng/syslog-ng/blob/master/dbld/images/helpers/pip_packages.manifest#L17.

I don't believe they are installed in the parent image. I might misunderstood something, though.

Reply:
You are right, I guess those are not installed at this point. I moved some dev tools out of the build images, IIRC because their installation was pretty fragile and not needed in those images.

But at the same time I didn't add an install_pip_packages into devshell's image, so I guess those have not been installed for about a year. But as I see you also added an inplace update for pip packages, why?

Reply:
Yes it has been like this for 10 months.

You mean this?
```
        devshell)
            python2 -m pip install --upgrade pip
            python3 -m pip install --upgrade pip
            ;;
```

Yes, the `python2` part is not really necessary right now, as `bionic` does that for us, I just wanted to make sure, if we change the to a `python3` based parent image, which will not update `python2-pip`, we still do that. I can remove it, if you want.